File: Programs/Java/calculator.java
import java.util.Scanner;

public class Calculator {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter first number: ");
        float first = scanner.nextFloat();

        System.out.print("Enter second number: ");
        float second = scanner.nextFloat();

        System.out.print("Enter operation (+, -, *, /): ");
        String operation = scanner.nextLine();

        switch (operation) {
            case "+":
                System.out.println("Result: " + (first + second));
                break;
            case "-":
                System.out.println("Result: " + (first - second));
                break;
            case "*":
                System.out.println("Result: " + (first * second));
                break;
            case "/":
                if (second != 0) {
                    System.out.println("Result: " + (first / second));
                } else {
                    System.out.println("Error: Division by zero!");
                }
                break;
            default:
                System.out.println("Unknown operation!");
                break;
        }

        scanner.close();
    }
}

File: Programs/C#/calculator.cs
using System;

public class Project
{
    public static void Main(string[] args)
    {
	    Console.Write("Enter first number: ");
	    float first = Convert.ToSingle(Console.ReadLine());
	    Console.Write("Enter second number: ");
	    float second = Convert.ToSingle(Console.ReadLine());
	    Console.Write("Enter operation(+,-,*,/): ");
	    string operation = Console.ReadLine();
	    
	    if (operation == "+") {
	        float result = first + second;
	        Console.WriteLine("Result: " + Convert.ToString(result));
	    }
	    
	    else if (operation == "-") {
	        float result = first - second;
	        Console.WriteLine("Result: " + Convert.ToString(result));
	    }
	    
	    else if (operation == "*") {
	        float result = first * second;
	        Console.WriteLine("Result: " + Convert.ToString(result));
	    }
	    
	    else if (operation == "/") {
	        float result = first / second;
	        Console.WriteLine("Result: " + Convert.ToString(result));
	    }
	    
	    else {
	        Console.WriteLine("Unknown operation!");
	    }
    }
} 


